CMakeLists.txt

```diff
@@ -679,6 +677,9 @@ if(EXECUTORCH_BUILD_PYBIND)
   if(EXECUTORCH_BUILD_KERNELS_QUANTIZED)
     target_link_options_shared_lib(quantized_ops_lib)
     list(APPEND _dep_libs quantized_kernels quantized_ops_lib)
+    if(TARGET quantized_ops_aot_lib)
+      list(APPEND _dep_libs quantized_kernels quantized_ops_aot_lib)
```

Comment:
This will add `quantized_kernels` to the list twice, since it was already added by line 681